seed_reader/test/design_tokens/dimensions_test.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:seed_reader/design_tokens/dimensions.dart';
+
+void main() {
+  test('dimensions', () {
+    expect(Dimensions.xSmall, 4);
+    expect(Dimensions.small, 8);
+    expect(Dimensions.regular, 16);
+    expect(Dimensions.large, 32);
```

Comment:
suggestion: this could be excluded. Validating screens aren't changing dimensions should be done through snapshot testing or [Golden_Toolkit](https://pub.dev/packages/golden_toolkit).